go.mod

```diff
@@ -38,8 +38,8 @@ require (
 	github.com/gorilla/mux v1.8.0
 	github.com/rancher/dynamiclistener v0.4.0-rc2
 	github.com/rancher/lasso v0.0.0-20240123150939-7055397d6dfa
-	github.com/rancher/rancher/pkg/apis v0.0.0-20240507213626-07f244b8be3a
-	github.com/rancher/rke v1.5.9-rc2
+	github.com/rancher/rancher/pkg/apis v0.0.0-20240611034301-19a4362e2243
```

Comment:
This may need to be bumped to a commit of https://github.com/rancher/rancher/pull/45852